classes/sync.php

```diff
@@ -538,6 +564,18 @@ protected function db_addslashes($text) {
         }
         return $text;
     }
+
+    /**
+     * Helper function to email site-admins on failure.
+     * @param string $message
+     */
+    protected function email_admins($message) {
+        global $DB, $CFG;
+        $users = $DB->get_records_list('user', 'id', explode(',', $CFG->siteadmins));
+        foreach ($users as $user) {
+            email_to_user($user, $user, 'tool_cohortdatbase error', $message);
```

Comment:
Wouldn't this be better coming from the support user?

Reply:
for upstream - probably yes... for Otago we need to spam all admins due to the severity of what might be occurring,... I'll look at some form of config for this instead to let the admin decide who should get it.

Reply:
I meant changing the `from` parameter to support user not who receives, might be less confusing than receiving an email to yourself from yourself?